docs/package.json

```diff
@@ -23,8 +22,6 @@
     "@vueuse/core": "^10.10.1",
     "@vueuse/nuxt": "^10.10.1",
     "nuxt": "npm:nuxt-nightly@latest",
```

Comment:
@harlan-zw is it intentional to have `npm:nuxt-nightly@latest` ?

Reply:
Just for testing purposes, happy to change it

Reply:
OK, I'll put this change in a separate PR ;)

Reply:
I responded in this PR.

- nuxt/scripts#100

Reply:
Hi @huang-julien , I've changed this.
Please check it out when you have time. :)